README.md

```diff
@@ -33,7 +48,8 @@ If you have Jupyter installed, you can create a notebook using IRkernel from the
 You can also start other interfaces with an R kernel:
 
 ```bash
-# “ir” is the kernel name installed by the above 'IRkernel::installspec()'
```

Comment:
R complained about "non ascii" during the build... Also, git diff didn't like them...